guides/common/modules/proc_applying-errata-to-hosts.adoc

```diff
@@ -1,49 +1,26 @@
 [id="applying-errata-to-hosts"]
 = Applying errata to hosts
 
-You can use the API to apply errata to a host, host group, or host collection.
-The following is the basic syntax of a PUT request:
-
-[options="nowrap", subs="+quotes,attributes"]
-----
-$ curl --header "Accept:application/json" \
---header "Content-Type:application/json" --request PUT \
---user _My_User_Name_:__My_Password__ \
---data _My_JSON_Formatted_Data_ https://_{foreman-example-com}_
-----
-
-You can browse the built-in API doc to find a URL to use for applying errata.
-You can use the {ProjectWebUI} to help discover the format for the search query.
-Navigate to *Hosts* > *Host Collections* and select a host collection.
-Go to *Collection Actions* > *Errata Installation* and notice the search query box contents.
-For example, for a Host Collection called _my-collection_, the search box contains `host_collection="my-collection"`.
-
-[id="exam-API_Guide-Applying_Errata_to_a_Host"]
-.Applying errata to a host
-
-This example uses the API URL for bulk actions `/katello/api/hosts/bulk/install_content` to show the format required for a simple search.
-
-Example request:
+You can use {Project} apply errata to hosts.
+ifndef::rest-api[]
+To use the API instead of the {ProjectWebUI}, see the xref:api-applying-errata-to-hosts[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Host Collections*.
+. Select a host collection.
+. On the *Actions* tab, click *Errata Installation*.
+. Follow the {ProjectWebUI} to install errata.
```

Comment:
"Follow the UI to install errata" is pretty vague for a procedure step.

If your PR covers only modularization, then my comment is probably outside the scope of this task. Do you think this justifies a bug?

Reply:
I believe that we've started to remove doc bloat **if** the WebUI is good and the user is guided by a wizard. This is one of these instances.

Reply:
This is a slippery slope. I want to discuss this with you offline.

Reply:
Can you please continue the conversation here (rather than offline)? I'm actually following this thread and am interested in the topic. (I agree with Max that if the steps are explained in the wizard/UI, duplicating them in the docs is not the best idea.)

Reply:
@maximiliankolb I checked the wizard in the web UI and yes, it works just fine on its own. Perhaps you could add its name, to help users check they landed on the right page in case of any confusion? How about something like this:

```suggestion
. Follow the *Content Host Errata Management* wizard in the {ProjectWebUI} to install errata.
```

Looking into the IBM style guide, it does include this in one of its examples:

> 1. [Step 1]
> 2. In the wizard, follow the instructions on each page.
> 3. [Step 3]

So it doesn't look like we're breaking any official convention with this approach.

Reply:
@asteflova Can you please provide me with a link? The ISG is not easy to search. Thanks.

Reply:
Found it. These examples appear in sections for using colons and periods. I would not assume that they apply to documenting wizards.

These are the examples that appear in the UI -> Wizards section:

In the Installation wizard, click the Servers tab.

The Configuration wizard opens.

The Configuration wizard is displayed.

In the “Printer setup” wizard, specify details about the printer that you want to use.

You can use the configuration wizard or command line to set preferences for the product.

Reply:
If the "installing errata" wizard is extremely brief, that might be a consideration.

Reply:
Re: my own reservations. This is something that I would prefer to discuss with our team.

Reply:
I'd rather not share a direct link to a closed resource. A full-text search of the pdf should get you where you need to be but just in case it doesn't: the example is in *Punctuation > Colons > Introductory text*. While I realize that this section is not centered around describing UI elements, I don't think it matters because the example is still there :shrug: Also, I didn't see anything that would discourage from this approach in *Technical elements > UI elements > Usage and highlighting for interface elements* (where wizards are actually described).

I'm not totally opposed to deviating from a style guide in cases when there is a reason. I didn't mean my comments to be a conversation stopper. As I mentioned, I looked at the wizard itself and reached the conclusion that no additional documentation support was necessary + I checked the style guide to make sure it supports this approach. If you disagree, I'd be happy to hear why :)

Reply:
It's interesting that this wizard is not documented anywhere. I was only able to find the [CLI procedure](https://docs.redhat.com/en/documentation/red_hat_satellite/6.16/html-single/managing_content/index#Applying_Errata_to_a_Host_Collection_content-management) for applying errata to a host collection. I checked the host management guide as well.